### DISCUSSION TITLE
How tu update date_scheduled on a order ?

### DISCUSSION THREAD

**Discussion by offlink-io** [swellstores/community#499] [Help] [answered]
Hello all,

I'am trying to set the `order.date_scheduled` but it is in read only:

```
errors: {
    date_scheduled: { message: 'Read-only field: date_scheduled', code: 'READONLY' }
  }

```

It's true that in the order model, it's put:

![image](https://github.com/swellstores/community/assets/128164601/60a2fc3c-1c84-47ff-911a-22cd13f113ed)

But in the backend order documentation, nothing about that:

![image](https://github.com/swellstores/community/assets/128164601/42c66e59-2209-4453-a583-9f140a93b8a3)

So, how i can update the field ?

Thx for your help

**Comment by logeshswell** [comment, marked as answer]
It is not recommended to update the value of `date_scheduled` field manually, since it is a read-only field and the default value is set using the formula.

But, may I know your use case to suggest you a better workaround?
